examples/step-40/step-40.cc

```diff
@@ -545,12 +545,17 @@ namespace Step40
   // bottleneck in parallelizing. In step-18, we have moved this step out of
   // the actual computation but shifted it into a separate program that later
   // combined the output from various processors into a single file. But this
-  // doesn't scale: if the number of processors is large, this may mean that
-  // the step of combining data on a single processor later becomes the
-  // longest running part of the program, or it may produce a file that's so
-  // large that it can't be visualized any more. We here follow a more
-  // sensible approach, namely creating individual files for each MPI process
-  // and leaving it to the visualization program to make sense of that.
+  // doesn't scale for several reasons: First, creating a single file per
+  // processor will overwhelm the filesystem with a large number of processors.
+  // Second, the step of combining all output files later can become the longest
+  // running part of the program, or it may produce a file that's so large that
+  // it can't be visualized easily any more.
+  //
+  // We here follow a more sophisticated approach that uses
+  // high-performance, parallel IO routines using MPI I/O to write to
+  // a small, fixed number of visualization files (here 8). We also
+  // generate a .pvtu record referencing these .vtu files, which can
+  // be opened directly in visualizatin tools like ParaView and VisIt.
```

Comment:
Actually, the text in lines 545-547 is no longer true. In step-18 we now just create one file per process. Can you replcae the sentence "In step-18, we have moved this step out of the actual computation but shifted it into a separate program that later combined the output from various processors into a single file." by the following:

In those two programs, we simply generate one output file per process. That worked because the parallel::shared::Triangulation cannot be used with large numbers of MPI processes anyway.

Then also remove the "Second, ..." text since it's no longer what we do.